crypto/asn1/tbl_standard.h

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
Change the main variable name as well? :)

Anyway, not so important. Perhaps I should bring up an addition to the style guide, though.